### PR TITLE
Changed function name from i2cBlockingTransmit to 12cTransmit and 12c…

### PR DESCRIPTION
**Pull request by ATGr8**
Changed function name from i2cBlockingTransmit to 12cTransmit and 12cBlockingRecive to i2ccRecieve

### CODE REVIEW

File: radsat-sk/framework/wrappers/RI2C.c [hunk 1]
```diff
@@ -40,7 +40,7 @@ uint32_t i2cInit() {
  * @param size Number of bytes to be written to the I2C slave
  * @return 0 for success, otherwise review I2C.h for return definition
  */
-uint32_t i2cBlockingTransmit(uint16_t slaveAddress, const uint8_t* data, uint16_t size) {
+uint32_t i2cTransmit(uint16_t slaveAddress, const uint8_t* data, uint16_t size) {
 
 	uint16_t err = I2C_write(slaveAddress, data, size);
```
Comment (ChrisAmaya) [inline radsat-sk/framework/wrappers/RI2C.c:45]: There is a pointer that is being expected in the I2C_write. Again, we can chat one day to talk about pointers and what not

File: radsat-sk/framework/wrappers/RI2C.c [hunk 2]
```diff
@@ -61,7 +61,7 @@ uint32_t i2cBlockingTransmit(uint16_t slaveAddress, const uint8_t* data, uint16_
  * @param size Number of bytes to be read from the I2C slave.
  * @return 0 for success, otherwise review I2C.h for return definition
  */
-uint32_t i2cBlockingRecieve(uint16_t slaveAddress, uint8_t* data, uint16_t size) {
+uint32_t i2cRecieve(uint16_t slaveAddress, uint8_t* data, uint16_t size) {
 
 	uint16_t err = I2C_read(slaveAddress, data, size);
```
Comment (ChrisAmaya) [inline radsat-sk/framework/wrappers/RI2C.c:67]: something we should really consider is that: Do we want "0" to be considered a success or would "1" make more sense.

**Review by ChrisAmaya — CHANGES_REQUESTED** [review]

File: radsat-sk/framework/wrappers/RI2C.c (hunk 2 above)
Comment (matthew-buglass) [reply radsat-sk/framework/wrappers/RI2C.c:67]: I think 0 should be considered a success. Even though many of our libraries use 1 for success, I believe that 0 is considered a success in standard C operations.

**Review by ChrisAmaya — APPROVED** [review]

**Review by matthew-buglass — APPROVED** [review]
Looks good

**Pull request merged**